jenkins-pipelines/longevity-100gb-4h-arm.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-100gb-4h.yaml", "configurations/arm_instance_types/Is4gen_4xlarge.yaml"]''',
+    availability_zone: 'c',
```

Comment:
is it true to all the regions we support?

Reply:
It's try for every region that supports ARM at all

Reply:
(That my account has access to. I can't talk for `us-gov-west-1`)

Reply:
In some of them, arm doesn't seem to be supported at all, so obviously there's nothing we can do about them.
Of those that do support them, some support regions a-c while other b-d